### PR TITLE
chore: Update URLs after moving projects

### PR DESCRIPTION
**Pull request by joeyparrish**

### CODE REVIEW

**Review by JulianDomingo — APPROVED** [review]

File: demo/index.html [hunk 1]
```diff
@@ -141,7 +141,7 @@ <h1 class="mdl-mega-footer__heading" localized-string="PROJECT_LINKS_HEADER"></h
               <ul class="mdl-mega-footer__link-list">
                 <li><a rel="noopener" target="_blank" href="../docs/api/index.html" localized-string="DOCUMENTATION"></a></li>
                 <li><a rel="noopener" target="_blank" href="https://www.apache.org/licenses/LICENSE-2.0" localized-string="LICENSE"></a></li>
-                <li><a rel="noopener" target="_blank" href="https://github.com/google/shaka-player" localized-string="SOURCE"></a></li>
+                <li><a rel="noopener" target="_blank" href="https://github.com/shaka-project/shaka-player" localized-string="SOURCE"></a></li>
                 <li><a rel="noopener" target="_blank" href="https://www.npmjs.com/package/shaka-player" localized-string="NPM"></a></li>
```
Comment (theodab) [inline demo/index.html:145]: Speaking of NPM, do we need to update the links on the NPM page?

File: lib/polyfill/mediasource.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ shaka.polyfill.MediaSource = class {
       shaka.polyfill.MediaSource.patchCastIsTypeSupported_();
     } else if (safariVersion) {
       // TS content is broken on Safari in general.
-      // See https://github.com/google/shaka-player/issues/743
+      // See https://github.com/shaka-project/shaka-player/issues/743
       // and https://bugs.webkit.org/show_bug.cgi?id=165342
```
Comment (theodab) [inline lib/polyfill/mediasource.js:43]: Oh hey, this issue was updated a few weeks ago!
They say it's fixed now, and I just ran the sample content in Safari and it seems to be fixed?
So maybe we can remove this. As another CL, obviously.

**Review by theodab — APPROVED** [review]

File: demo/index.html (hunk 1 above)
Comment (joeyparrish) [reply demo/index.html:145]: No, because that isn't namespaced the same way our GitHub repos are.

File: lib/polyfill/mediasource.js (hunk 2 above)
Comment (joeyparrish) [reply lib/polyfill/mediasource.js:43]: Sounds good.  We should do a round of TODO cleanup before we push out v4.

**Pull request merged**